### ISSUE TITLE
Druggability project

### ISSUE THREAD

**Issue by giadefa** [multiscalelab/master2016#2]
In this project the aim is to create a python package that it is similar to mdmix but uses the adaptive method to compute all poses better than -2 kcal/mol. The input is the pdb of the protein and the output is a molecule object with all the best poses (-2kcal/mol) for all the probes. You need at least 4 probes. Hyrophobic, polar, charged positive, negative. Use the gpu machines to test the program on trypsin and write a report.

**Comment by alejandrovr** [comment]
Should all probes be considered at the same time? Like creating a Molecule object and append the 4 probes to it? Or should we consider each probe independently?

**Comment by giadefa** [comment]
I don't understand what you mean. At what stage? The runs should be
independent. You can merge the final results together for visualization

**Comment by alejandrovr** [comment]
Ok, that solves my question. Thank you.

**Comment by alejandrovr** [comment]
Could we meet this week to solve some questions?

**Comment by alejandrovr** [comment]
We have made the binding analysis for the interaction between ethanol (polar ligand) and trypsin (the other 3 still running), and we have obtained a few macrostates. We have ploted their free energies with kin.plotRates(rates=('g0eq')) and we have obtained this:
![plotg0](https://cloud.githubusercontent.com/assets/16488180/16377111/74383c02-3c64-11e6-8791-04d29b50ab73.png)

We have some doubts:

1) Is this energy the one we should use to filter the poses using the 2 kcal/mol threshold?

2) We are able to extract each macrostate as a Molecule object with getStates(). Is this the molecule object which we have to return as the output of the project? or is it the individual poses that compose the macrostate?

3)Could we obtain the information that the plot in the image in form of a dictionary? Like: Macrostate 0 g0eq: -2.9 kcal/mol, macrostate2 g0eq: -2 kcal/mol, etc.

Would it be possible to meet today or tomorrow?. We are a bit short of time. Sorry for the long post.
